controllers/team.controller.js

```diff
@@ -0,0 +1,61 @@
+const Team = require('../models').Team;
+
+const defaultResponse= require('../utils/defaultResponse');
+const constants= require('../utils/constants');
+const responseStatus= require('../utils/responseStatus');
+
+exports.getTeam = async (req, res) => {
+    try {
+        const team = await Team.findAll();
+        if (team) {
+            defaultResponse().success(constants.DATA_RETRIEVED, team, res, responseStatus.SUCCESS);
+        } else
+           defaultResponse().success(constants.DATA_NOT_FOUND, null, res, responseStatus.SUCCESS);
+    } catch (exception) {
+        defaultResponse().error({message:exception.message}, res, responseStatus.ERROR)
```

Comment:
Please ensure the message and error sent here as a response is known and has an error code.